app/components/note-list-item/note-list-item.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class NoteListItem extends Component{
+  render() {	
+    return (
+      <div className="card"> 
+		{this.props.note}
```

Comment:
Indentation issue. By card i meant a card component from one of the react UI frame works.
For eg: [ANT](https://ant.design/components/card/), [React-Bootstrap](https://react-bootstrap.github.io/components/panel/)

Reply:
okay, i will work on it later today im a bit busy at the moment 
hopefully  i will send another PR in 9 hours sorry about that